src/test/rustdoc-gui/anchors.goml

```diff
@@ -1,17 +1,17 @@
 goto: file://|DOC_PATH|/test_docs/struct.HeavilyDocumentedStruct.html
-
```

Comment:
nit: Blank line was removed (accidentally?).

Reply:
Ah good point, I'll put it back.